### PR TITLE
[PW-7810] Create templates for small services

### PR DESCRIPTION
**Pull request by antolo-arch**
**Description**
Templates for smaller services

**Tested scenarios**
Tested locally, generated services will be tested in different PRs

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,8 @@ coverage:
 
 generator:=python
 openapi-generator-cli:=java -jar build/openapi-generator-cli.jar
-services:=balancePlatform binlookup checkout dataProtection legalEntityManagement management payments payouts platformsAccount platformsFund platformsHostedOnboardingPage platformsNotificationConfiguration recurring storedValue terminalManagement transfer
+services:=balancePlatform checkout legalEntityManagement management payments payouts platformsAccount platformsFund platformsHostedOnboardingPage platformsNotificationConfiguration terminalManagement transfer
+smallServices:= recurring binlookup dataProtection storedValue balanceControlService
```
Comment (wboereboom) [inline Makefile:14]: could be alphabetic order 🤷

**Review by wboereboom — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (jillingk) [reply Makefile:14]: 😆

File: templates/api.mustache [hunk 2]
```diff
@@ -1,4 +1,3 @@
-{{>partial_header}}
```
Comment (jillingk) [inline templates/api.mustache:1]: Why is the partial header removed? Does the api-small class extend the api.mustache class or smth?

**Review by jillingk — APPROVED** [review]

File: templates/api.mustache (hunk 2 above)
Comment (antolo-arch) [reply templates/api.mustache:1]: not to include the header in all the classes as it's generated for each subclass, small one is generated once, thus the header stays there

File: Makefile (hunk 1 above)
Comment (antolo-arch) [reply Makefile:14]: Will fix that in the next PR!

File: templates/api.mustache (hunk 2 above)
Comment (jillingk) [reply templates/api.mustache:1]: ahh okay, so only the init files in the big services have that header?

**Pull request merged**